File: src/Photoportfolio/wwwroot/lib/spa/core/domain/photo.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var Photo = (function () {
    function Photo(id, title, uri, albumId, albumTitle, author, rating, dateUploaded) {
        this.Id = id;
        this.Title = title;
        this.Uri = uri;
        this.AlbumId = albumId;
        this.AlbumTitle = albumTitle;
        this.Author = author;
        this.Rating = rating;
        this.DateUploaded = dateUploaded;
    }
    return Photo;
}());
exports.Photo = Photo;

File: src/Photoportfolio/Entities/Photo.cs
using System;
using System.Collections.Generic;

namespace Photoportfolio.Entities
{
    public class Photo : IEntityBase
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Uri { get; set; }
        public int AlbumId { get; set; }
        public float Rating { get; set; }
        public DateTime DateUploaded { get; set; }
        public virtual Album Album { get; set; }
        public virtual ICollection<UserFeedback> Feedbacks { get; set; }
    }
}
